packages/apps/lightning-ui-docs/.storybook/addons/components/ColorRow.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import { ColorControl } from '@storybook/blocks';
+import { updateComponentValue } from './componentUtils';
+import { TableRow } from '../components';
+
+/**
+ *
+ * @returns component style row with color control
+ */
+export default function ColorRow({
+  version,
+  styleProp,
+  defaultValue,
+  componentName,
+  updateGlobals
+}) {
+  const [fieldValue, setValueState] = useState(defaultValue);
+
+  return (
+    <TableRow
+      label={styleProp}
+      control={
+        <ColorControl
+          name={styleProp}
+          key={`Color-${componentName}-${styleProp}-${version}`}
```

Comment:
For the same reasoning in my comment in NumberRow, I don't believe `key` is needed here.

Reply:
Yea. good catch, I  forgot to remove this in all my rearranging